src/js/fs.js

```diff
@@ -275,6 +275,76 @@ fs.readFileSync = function(path) {
 };
 
 
+fs.writeFile = function(path, data, callback) {
+  checkArgString(path);
+  checkArgBuffer(data);
+  checkArgFunction(callback);
+
+  var fd;
+  var len;
+  var bytesWritten;
```

Comment:
I cannot find explicit rule in IoT coding style guide. Although I think it is also the problem of taste. (there was a project that prohibits declaring several variables in one line. I can't remember the project name.). I will put them in one line. 

In addition, I will fix the year information.